### ISSUE TITLE
Where to move `class downloader` from `utilities/gdrive.py`?

### ISSUE THREAD

**Issue by orsk-moscow** [orsk-moscow/utilities#42]
### Description

```markdown
Where to move `class downloader` from `utilities/gdrive.py`?
```

### Additional context

```plain text
-
```

### Urgency

Issue can be postponed

### Priority Prod Data.

No

### Priority of a Time.

No

### Priority of a Money 1.

No

### Priority of a Money 2.

No

### Blocks

-

### Blocked by

-

### Size of related Repositories

utilities

### Size of Pipelines

Dict, Inbox

### Size of documentation

Existing documentation

### Size of Test coverage

Unit tests already cover the issue, Unit tests need to be updated, Unit tests need to be written from scratch

### Acceptance criteria

```markdown
1. All unit tests passed. 
2. `class downloader` moved and refactored.
```

### Specific test cases

```markdown
There are no Specific test cases.
```
